File: 2020/1/first.py
numbers = []
with open("data.txt") as f:
    for line in f:
        numbers.append(int(line))
        
    for i, a in enumerate(numbers):
        for j, b in enumerate(numbers[i+1:]):
            if a + b == 2020:
                print(a, b, a*b)

File: 2020/1/first.cpp
#include <fstream>
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int main()
{
    string line;
    vector<int> data;
    ifstream myfile("data.txt");
    if (myfile.is_open()) {
        while (getline(myfile, line)) {
            int a = stoi(line);
        }
        myfile.close();
        cout << data.length();
    }
}